src/apps/session/SearchItem.tsx

```diff
@@ -42,51 +43,53 @@ const SearchItem = (props: Props) => {
         <TranslationContext.Provider
           value={{ lang, t }}
         >
-          <MapSearchContextProvider
+          <SearchConfigContextProvider
```

Comment:
@camdendotlol - I noticed a bug here. I remember you doing some work around this when implementing the non-map searches, so I just wanted to make sure I'm making the correct changes here.